diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 523b2cb..ea1dd25 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -33,8 +33,7 @@ set_target_properties(cmark PROPERTIES
   PDB_NAME libcmark
   POSITION_INDEPENDENT_CODE YES
   # Include minor version and patch level in soname for now.
-  SOVERSION ${PROJECT_VERSION}
-  VERSION ${PROJECT_VERSION})
+  )
 if(NOT BUILD_SHARED_LIBS)
   target_compile_definitions(cmark PUBLIC
     CMARK_STATIC_DEFINE)
